src/metatrain/experimental/mace/model.py

```diff
@@ -0,0 +1,739 @@
+import logging
+import warnings
+from pathlib import Path
+from typing import Any, Dict, List, Literal, Optional
+
+import mace.modules as mace_modules
+import metatensor.torch as mts
+import torch
+from e3nn import o3
+from e3nn.util import jit
+from mace.modules import MACE
+from metatensor.torch import Labels, TensorBlock, TensorMap
+from metatensor.torch.operations._add import _add_block_block
+from metatomic.torch import (
+    AtomisticModel,
+    ModelCapabilities,
+    ModelMetadata,
+    ModelOutput,
+    NeighborListOptions,
+    System,
+)
+
+from metatrain.utils.abc import ModelInterface
+from metatrain.utils.additive import CompositionModel
+from metatrain.utils.data import DatasetInfo, TargetInfo
+from metatrain.utils.dtype import dtype_to_str
+from metatrain.utils.metadata import merge_metadata
+from metatrain.utils.scaler import Scaler
+from metatrain.utils.sum_over_atoms import sum_over_atoms
+
+from .documentation import ModelHypers
+from .modules.finetuning import apply_finetuning_strategy
+from .modules.heads import MACEHeadWrapper, NonLinearHead
+from .modules.scale_shift import FakeScaleShift
+from .utils.mts import (
+    e3nn_to_tensormap,
+    get_e3nn_target_info,
+    get_samples_labels,
+    target_info_to_e3nn_irreps,
+)
+from .utils.structures import create_batch
+
+
+class MetaMACE(ModelInterface[ModelHypers]):
+    """Interface of MACE for metatrain."""
+
+    __checkpoint_version__ = 1
+    __supported_devices__ = ["cuda", "cpu"]
+    __supported_dtypes__ = [torch.float64, torch.float32]
+    __default_metadata__ = ModelMetadata(
+        references={
+            "architecture": [
+                "https://arxiv.org/abs/2205.06643",
+                "https://openreview.net/forum?id=YPpSngE-ZU",
+            ]
+        }
+    )
+
+    # Attributes of the model. We can't uncomment these descriptions because
```

Comment:
maybe you can remove the docstring duplicates? And only leave the type annotations

Reply:
my bad, I read them too fast and though they were hypers. all good!